java/de/yessoft/android/mohammad/services/MessageReceiver.java

```diff
@@ -0,0 +1,60 @@
+package de.yessoft.android.mohammad.services;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Intent;
+import android.media.RingtoneManager;
+import android.net.Uri;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+import de.yessoft.android.mohammad.R;
+import de.yessoft.android.mohammad.ui.MainActivity.MainActivity;
+
+public class MessageReceiver extends FirebaseMessagingService {
+    // private static final String TAG = "ML_MessageReceiver";
+
+    private static final int REQUEST_CODE = 1;
+    private static final int NOTIFICATION_ID = 6578;
+
+    public MessageReceiver()
+    {
+        super();
+    }
+
+    @Override
+    public void onMessageReceived(RemoteMessage remoteMessage) {
+        super.onMessageReceived(remoteMessage);
+
+        RemoteMessage.Notification notification = remoteMessage.getNotification();
+
+        if (notification != null){
+            String title = remoteMessage.getNotification().getTitle();
+            String message = remoteMessage.getNotification().getBody();
+            showNotifications(title, message);
+        }
+    }
+
+    private void showNotifications(String title, String msg) {
+        Intent i = new Intent(this, MainActivity.class);
```

Comment:
variable names

Reply:
yep @Kenan-Hussein , i was at fault too, so we need to keep that in mind

Reply:
@MickSawy3r  Sure.